backend/api/serializers/group_serializer.py

```diff
@@ -13,52 +13,62 @@
 
 
 class GroupSerializer(serializers.ModelSerializer):
-    project = ProjectSerializer(
-        read_only=True,
+    project = serializers.HyperlinkedIdentityField(
+        view_name="project-detail"
     )
 
     students = serializers.HyperlinkedIdentityField(
         view_name="group-students",
         read_only=True,
     )
 
+    occupation = serializers.SerializerMethodField()
+
     submissions = serializers.HyperlinkedIdentityField(
         view_name="group-submissions",
         read_only=True,
     )
 
-    class Meta:
-        model = Group
-        fields = "__all__"
+    def get_occupation(self, instance: Group):
+        """Get the number of students in the group"""
+        return instance.students.count()
 
-    def to_representation(self, instance):
+    def to_representation(self, instance: Group):
+        """Convert the group to a JSON representation"""
         data = super().to_representation(instance)
 
         user = self.context["request"].user
         course_id = instance.project.course.id
 
         # If you are not a student, you can always see the score
-        if is_student(user):
+        if is_student(user) and not is_teacher(user) and not is_assistant(user) and not user.is_staff:
```

Comment:
Kan het nu niet het geval zijn dat een assistent, die ook een student is, de scores kan zien van alle groepen van een project van een vak waar die geen assistent van is, maar wel een student van is?

Reply:
Yes